binderhub/tests/test_build.py

```diff
@@ -176,7 +179,8 @@ async def test_build_fail(app, needs_build, needs_launch, always_build):
     assert failed_events > 0, "Should have seen phase 'failed'"
 
 
-@pytest.mark.asyncio(timeout=120)
+@pytest.mark.timeout(120)
+@pytest.mark.asyncio
```

Comment:
Was pytest.mark.asyncio needed? We have a lot of jupyterhub repos with an automatic detection mode, making all async test functions assumed this decorator

Reply:
I think this may make it not needed, but also not a problem to have explicit

https://github.com/jupyterhub/binderhub/blob/58bb7dba89baf739c4015a6afc59d6ec3c8d59d7/pyproject.toml#L34

Reply:
I'll take it out